source/2020-02-21-the-ember-times-issue-136.md

```diff
@@ -35,13 +36,21 @@ A **new** version of Ember is here! [Ember 3.16](https://blog.emberjs.com/2020/0
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [2020 Ember Community Survey 📝](https://tilde.wufoo.com/forms/2020-emberjs-community-survey/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+It’s that time of year again 😀, the 6th annual official 2020 Ember Community Survey is here! Once again we would love your help to learn about who is in the Ember community and how they work with the framework.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+And there's good news! This year the survey has been considerably slimmed down from previous years. So it should be easier than ever to make a contribution to our community by filling out the survey.
+
+Last year over 1200 people participated, and thankfully all that participation and your participation over the years has resulted in enough input to make the process more efficient. So let’s keep that participation going! 🎉
```

Comment:
I think I understand why you added the middle "thankfully...more efficient" (it ties into the previous paragraph). I'm not sure if it's necessary to mention, though.

If not, maybe we can shorten the paragraph to:

```suggestion
Last year over 1200 people participated. Let’s keep that momentum going! 🎉
```

We could even move these two sentences to the end of the previous paragraph.

Reply:
Thanks for the review @ijlee2! I wanted to make sure that we link participation to the reduced length so that it incentivizes folks to continue to contribute. I reworked the thoughts there to hopefully make it flow a little better.